tests/local/local_test.go

```diff
@@ -102,34 +94,22 @@ func TestSetup(t *testing.T) {
 
 func testModules(t *testing.T) {
 	t.Run("Dashboard", testLeadDashboard)
-	t.Run("SDM", testLeadSdm)
-	t.Run("KubeResourceReport", common.KubeResourceReportTest)
-	t.Run("ExternalDNS", common.ExternalDnsTest)
-	t.Run("KubeDownscaler", common.KubeDownscalerTest)
-	t.Run("K8sSpotTerminationHandler", common.K8sSpotTerminationHandlerTest)
-	t.Run("KubeJanitor", common.KubeJanitorTest)
-	t.Run("MetricsServer", common.MetricsServerTest)
+
+	//t.Run("SDM", testLeadSdm)
+	//t.Run("KubeResourceReport", common.KubeResourceReportTest)
+	//t.Run("ExternalDNS", common.ExternalDnsTest)
+	//t.Run("KubeDownscaler", common.KubeDownscalerTest)
+	//t.Run("K8sSpotTerminationHandler", common.K8sSpotTerminationHandlerTest)
+	//t.Run("KubeJanitor", common.KubeJanitorTest)
+	//t.Run("MetricsServer", common.MetricsServerTest)
+	//t.Run("PrometheusOperator", common.TestPrometheusOperator)
```

Comment:
did you intend to leave this commented out?

Reply:
Definitely did not, I think my formatting fix I pushed up inadvertently included this version from testing the `lead-dashboard` module. Just pushed up a better copy, sorry about that.